### PR TITLE
Add 2 new tests for DATM-MOM6-CICE6 application

### LINKED ISSUE #260: Adde a debug test, a 6-hr test, a 12-hr test, and a restart test for the DATM-MOM6-CICE6 application
## Description
A debug test, a 6-hr test, a 12-hr test, and a restart test will be added to the DATM-MOM6-CICE6 regression test system.

## Solution
The following three new tests will use 1-degree MOM6 and CICE6. The atmospheric forcing data will be from GEFS:

1.  6-hr test
2. 12-hr test
2. restart test. The model will run from hour 6 to hour 12.
3. debug test.

Related issue:
#281

### PR DESCRIPTION
**Pull request by binli2337**
## Description
The following 2 new regression tests are added for the DATM-MOM6-CICE6 application:
datm_restart_cfsr
datm_debug_cfsr

Bugs in default_vars.sh, fv3_conf/cpld_datm_cfsr.IN and fv3_conf/cpld_datm_gefs are fixed.

### Issue(s) addressed
- fixes #260 

## Testing
Baseline for existing tests will not be changed. New baseline directories for the 2 new tests will be added.

Co-author: DeniseWorthen

### CODE REVIEW

File: tests/rt.conf [hunk 1]
```diff
@@ -252,20 +252,18 @@ RUN     | cpld_debug
 #######################################################################################################################################################################################
 # Data Atmosphere tests
 #######################################################################################################################################################################################
+COMPILE | DATM=Y S2S=Y                                                                                                        | standard | hera.intel orion.intel wcoss_dell_p3 | fv3 |
```
Comment (DeniseWorthen) [inline tests/rt.conf:256]: Is there a reason you've removed the comment-ed out Cheyenne lines?

File: tests/tests/datm_restart_cfsr [hunk 2]
```diff
@@ -0,0 +1,26 @@
+#
```
Comment (DeniseWorthen) [inline tests/tests/datm_restart_cfsr:1]: I'm surprised you didn't implement the restart test using a checkpoint restart which would have only required one additional test---using restarts from hour 12 of the 1d run, integrating for 12h and comparing to the existing 1d run.

File: tests/rt.conf (hunk 1 above)
Comment (binli2337) [reply tests/rt.conf:256]: It will be added.

**Comment by climbfuji** [comment]
@binli2337 can you please let us know as soon as the PR is ready so that we can create baselines and run tests on all platforms? Thanks!

**Review by DeniseWorthen — APPROVED** [review]
Changes look correct to me for using checkpoint restart.

**Comment by climbfuji** [comment]
Is this PR ready for review? Please add the appropriate labels for the PR, if possible.

**Comment by DusanJovic-NOAA** [comment]
Please change the title and description of this PR. Only 2 new tests are added.

**Review by junwang-noaa — APPROVED** [review]
This commit only adds DATM tests, there is no impact on other application tests (global, regional and coupled). The RT has only run on dell, hera and orion that support DATM.  New DATM debug test results are added to current baseline develop-20201215 on these platforms.

**Review by climbfuji — APPROVED** [review]

**Pull request merged**